### PR TITLE
Add eslint & prettier to server

### PR DESCRIPTION
**Pull request by shiggabyte**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,30 +1,25 @@
-const express = require("express");
-const morgan = require("morgan");
-
+const express = require('express');
+const morgan = require('morgan');
+// database connection
+/* const db =  require("./config/database"); */
```
Comment (abdulfataiaka) [inline server/server.js:4]: ☝️

**Review by abdulfataiaka — APPROVED** [review]
LGTM

But let's remove all commented (unused) codes anywhere you see them before merging this PR. I have used the point up emoji (☝️) to indicate one i saw, though here are others

File: server/server.js (hunk 1 above)
Comment (shiggabyte) [reply server/server.js]: the commented codes have all been removed. may i merge?

**Comment by abdulfataiaka** [comment]
Knock yourself out 😄

File: server/server.js (hunk 1 above)
Comment (abdulfataiaka) [reply server/server.js]: Knock yourself out 😄

**Pull request merged**